README.md

```diff
@@ -53,48 +60,32 @@ Along with regressions on each site independent of one another and onto two site
 ```
 --pheno_file
 ```
-Input a file with phenotype values corresponding to each sequence in the sequence file.
+Input a file with phenotype values corresponding to each sequence in the sequence file. If you have a .xlsx or .csv file, do NOT use this flag (more details above in the **Gather the Input Files Needed** section).
```

Comment:
minor comment - you can link to this section by [Gather ..][https://github.com/snafees/ortho_seqs/blob/e939899733f080bebd41ea05a068fc9c278b99cf/README.md#gather-the-input-files-needed]

Reply:
Good idea, added!